src/ray/protobuf/experimental/instance_manager.proto

```diff
@@ -17,29 +17,15 @@ option cc_enable_arenas = true;
 
 package ray.rpc.autoscaler;
 
-enum RayNodeKind {
-  UNKNOWN = 0;
-  HEAD = 1;
-  WORKER = 2;
-}
-
-// A node type that's available for the cluster.
-message InstanceType {
-  // the name of the instance type. e.g. "m4.large"
-  string type_name = 1;
-  RayNodeKind node_kind = 2;
-  // avaialble resources on the node. such as {"CPU": 4, "GPU": 1}
-  map<string, double> resources = 3;
-}
-
-message GetAvailableInstanceTypesRequest {}
-
-message GetAvailableInstanceTypesResponse {
-  repeated InstanceType instance_types = 1;
-  // number of instances that are available for starting.
-  // this can change if the cloud provider has a limit on
-  // number of instances that can be started.
-  int64 available_instances = 2;
```

Comment:
These are not really needed.

Reply:
hmm, does autoscaler need to know this information?

Reply:
autoscaler would need this info to talk to the scheduler.

Reply:
Autoscaler could get the available types info (e.g. resources) from a few places: 
1. autoscaling configs 
2. node providers 
3. instance managers (most likely need to ask node providers) 

I feel 2 is the most intuitive places.